### PR TITLE
refactor(governance): converge final support universe claim

### PR DESCRIPTION
**Pull request by jamesryancooper**
## What
Converges Octon's active claim from the earlier bounded-era posture to a final admitted-universe posture, regenerates the active closure/disclosure bundle, and archives the two exploratory proposal packets after execution.

## Why
The repo had already implemented the bounded certified live universe, but the active governance and disclosure surfaces still carried staged/non-final claim semantics for modeled critical surfaces. This change resolves those surfaces into explicit non-live exclusions, updates the generators and validators to enforce the final posture, and leaves one truthful active claim surface.

No-Issue: final support-universe convergence and packet execution closeout

## How
Updated support-target governance, support dossiers/admissions, runtime pack admissions, constitutional/workspace claim metadata, and closure/disclosure surfaces so the live universe is explicitly admitted and all remaining modeled critical surfaces are explicit non-live exclusions. Then regenerated the active release bundle and closure projections and archived the executed packet inputs under `/.octon/inputs/exploratory/proposals/.archive/architecture/`.

## Profile Selection Receipt
- Semantic version source(s): `/.octon/framework/constitution/charter.yml`, `/.octon/instance/charter/workspace.yml`
- Release state (`pre-1.0` or `stable`): `pre-1.0`
- `change_profile` (`atomic` or `transitional`): `atomic`
- Hard-gate facts (downtime, coordination, migration/backfill, rollback, blast radius, compliance): no downtime; one active claim surface; authored/governance and generated release artifacts must move together; rollback is branch revert to prior bounded-era claim bundle
- Tie-break status: none
- `transitional_exception_note` (required when `pre-1.0` + `transitional`): n/a

## Implementation Plan
- Workstreams and scope: converge support-target claim mode; resolve staged modeled surfaces into explicit non-live exclusions; update generators/validators; regenerate release bundle and effective projections; archive executed packets and retarget the execution prompt
- Public interfaces/types/contracts affected: support-target declaration/schema, closure/disclosure artifacts, pack admissions/registry, contract family receipt refs, claim-truth conditions
- Test scenarios: support-target normalization, live-claim validation, capability truthfulness, retirement readiness, and global closure assertion
- Assumptions/defaults: keep exactly two supported tuples live; keep non-live modeled surfaces explicit rather than widening support without new proof

## Impact Map (code, tests, docs, contracts)
- Code: generator and validator scripts under `/.octon/framework/assurance/**`
- Tests: validation scripts and closure assertions consume the new final-state semantics
- Docs: browser/api pack READMEs, governance disclosure docs, workspace/charter metadata, execution prompt
- Contracts: support-target schema/declaration, constitutional family receipts, closure/disclosure manifests, pack admissions

## Compliance Receipt
- [x] Selected exactly one execution profile before planning/implementation.
- [x] Applied release-maturity gate from semantic version.
- [x] Enforced pre-1.0 default (`atomic`) unless hard gates required `transitional`.
- [x] Included `transitional_exception_note` when required.
- [x] Included tie-break escalation when applicable.
- [x] Updated impacted contracts/docs/tests.
- [x] Honored charter change-control constraint (no direct principles charter edits without override evidence).

## Exceptions/Escalations
none

## Tradeoffs
The final live claim remains intentionally narrow: only the proven repo-shell/repo-local-governed English/reference-owned tuples are live. Modeled host, pack, locale, model-adapter, and boundary-sensitive surfaces remain explicit non-live exclusions instead of being widened speculatively.

## Testing
Automated:
- `bash .octon/framework/assurance/governance/_ops/scripts/validate-support-target-normalization.sh`
- `bash .octon/framework/assurance/runtime/_ops/scripts/validate-support-target-live-claims.sh`
- `bash .octon/framework/assurance/governance/_ops/scripts/validate-capability-truthfulness.sh`
- `bash .octon/framework/assurance/runtime/_ops/scripts/validate-retirement-registry.sh`
- `bash .octon/framework/assurance/runtime/_ops/scripts/validate-global-retirement-closure.sh`
- `bash .octon/framework/assurance/governance/_ops/scripts/assert-unified-execution-closure.sh`

Regeneration:
- `bash .octon/framework/assurance/runtime/_ops/scripts/generate-release-bundle.sh 2026-04-06-target-state-closure-provable-closure`
- `bash .octon/framework/assurance/runtime/_ops/scripts/generate-closure-projections.sh 2026-04-06-target-state-closure-provable-closure`

## Rollout
n/a

## Convivial Purpose Check
- [x] Feature expands genuine user capability (not synthetic engagement).
- [x] Attention and interruption behavior are justified and user-controllable.
- [x] No manipulative patterns or dark-pattern mechanics are introduced.
- [x] Data collection/extraction risk is minimal and explicitly justified.

## Risk Rubric
- Risk class: [ ] Trivial [ ] Low [x] Medium [ ] High
- Rollback plan: revert this branch and regenerate the prior bounded-era release bundle
- Rollback handle: commit `ef3e7a16f`
- Flags changed (name, owner, expiry, rollout): none
- Autonomy eligibility: [ ] autonomy:auto-merge [x] autonomy:no-automerge

## Contracts and Threat Model
- OpenAPI/JSON-Schema changes: support-target schema wording/property rename for resolved non-live surfaces
- Threat model update/link: no separate threat-model document updated; authority/fail-closed posture preserved and validated

## Observability and Performance
- Traces/logs/metrics for changed flows: closure/disclosure evidence regeneration only
- Representative traces for high risk changes: closure gate report, support-universe coverage, proof-plane coverage, cross-artifact consistency, claim-drift report
- Performance or bundle impact: negligible

## License and Provenance
- New dependencies and licenses: none
- Generated code/templates provenance notes: release bundle, closure projections, and migration receipts regenerated from canonical repo surfaces

## Checklist
- [x] Requirements met; edge cases handled
- [x] Security reviewed (authz, input validation, secrets)
- [x] Tests added or updated
- [x] Observability updated (logs, metrics, traces) if needed
- [x] No speculative abstractions or unnecessary complexity
- [x] Conventions followed; no drift introduced
- [x] Non-obvious decisions documented (comments, ADR)
- [ ] All review conversations resolved

## Screenshots / Notes
No visual changes. The archived exploratory packets remain non-authoritative inputs and were moved to the archive path after execution.

### CODE REVIEW

File: .octon/instance/governance/support-targets.yml [hunk 1]
```diff
@@ -66,22 +66,22 @@ tiers:
       description: "Consequential repository work that is admitted only inside the bounded claim envelope when mission-backed and fully evidenced."
     - id: "boundary-sensitive"
       label: "boundary-sensitive"
-      default_route: "escalate"
-      description: "Boundary-sensitive work that remains outside the live claim and is stage-only until proof and recovery posture are fully closed."
+      default_route: "deny"
```
[the remark this answers is not in the document]
Comment (jamesryancooper) [reply .octon/instance/governance/support-targets.yml:69]: Patched. `support-targets.schema.json` now allows `deny` for workload-tier `default_route`, which matches the converged non-live posture in `support-targets.yml`. I also pushed the follow-up commit `8cc5b8505` to rerun checks.
